File: C_Programming/Practise_Programs/PP_060.c
/*
==============================================================================
 Program     : Display Array Elements Using a Loop

 Description : Demonstrates how to traverse an integer array and display all
               of its elements using a for loop.

 Author      : Shubham Gadhe
==============================================================================
*/

#include<stdio.h>

int main()
{
    int Arr[5] = {10,20,30,40,50};

    int iCnt = 0;

    // Traverse the array and display each element.
    for(iCnt = 0; iCnt < 5 ; iCnt++)
    {
        printf("%d\n", Arr[iCnt]);
    }   

    return 0;   
}

/*
==============================================================================
 Sample Output
------------------------------------------------------------------------------
 10
 20
 30
 40
 50
==============================================================================
*/

File: Java_Programming/Practise_Programs/PP_060.java
//////////////////////////////////////////////////////////////////
//
//  File Name         : PP_060
//
//  Description       : This program accepts the number of rows
//                      and columns from the user and displays
//                      numbers from 1 to the specified number
//                      of columns in each row.
//
//  Author            : Shubham Somanath Gadhe
//  Date              : 07/08/2026
//
//  Time Complexity   : O(r × c)
//  Space Complexity  : O(1)
//
//////////////////////////////////////////////////////////////////

import java.util.*;

class PP_060
{
    //////////////////////////////////////////////////////////////////
    //
    //  Function Name    : Display()
    //
    //  Description      : It is used to display numbers
    //                     from 1 to the specified number
    //                     of columns in each row.
    //
    //  Author           : Shubham Somanath Gadhe
    //  Date             : 07/08/2026
    //
    //////////////////////////////////////////////////////////////////

    public static void Display(int iRow, int iCol)
    {
        int i = 0, j = 0;

        for(i = 1; i <= iRow; i++)
        {
            for(j = 1; j <= iCol; j++)
            {
                System.out.print(j + "\t");
            }

            System.out.println();
        }
    }

    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue1 = 0, iValue2 = 0;

        System.out.println("Enter the number of Rows: ");
        iValue1 = sobj.nextInt();

        System.out.println("Enter the number of Columns : ");
        iValue2 = sobj.nextInt();

        Display(iValue1, iValue2);
    }
}

//////////////////////////////////////////////////////////////////
//
//  Sample Input  :
//      Enter the number of Rows :
//      4
//
//      Enter the number of Columns :
//      4
//
//  Sample Output :
//
//      1    2    3    4
//      1    2    3    4
//      1    2    3    4
//      1    2    3    4
//
//////////////////////////////////////////////////////////////////